app/models/foreman_google/google_compute.rb

```diff
@@ -30,9 +30,11 @@ def status
     end
 
     def start
+      @client.start(@zone, @identity)
```

Comment:
should we add guard for `if persisted?` :)
Not strictly necessary, but would be good manners IMHO

Reply:
Yeah probably, should we do it also for the `stop` method?